src/pybind/nnet3/nnet_nnet_pybind.cc

```diff
@@ -19,31 +19,90 @@
 
 #include "nnet3/nnet_nnet_pybind.h"
 
-#include "nnet3/nnet-nnet.h"
 #include "nnet3/nnet-component-itf.h"
+#include "nnet3/nnet-convolutional-component.h"
+#include "nnet3/nnet-nnet.h"
+#include "nnet3/nnet-normalize-component.h"
+#include "nnet3/nnet-simple-component.h"
 
 using namespace kaldi;
 using namespace kaldi::nnet3;
 
+template <typename... Args>
```

Comment:
You do not need to redefine it here.

It is already defined in
https://github.com/kaldi-asr/kaldi/blob/cc63cca97ecd4b54d1b3f1dd9994a3117c999404/src/pybind/kaldi_pybind.h#L30-L31